client/components/AdoptionForm.js

```diff
@@ -0,0 +1,98 @@
+import React,{useState} from "react"
+
+const AdoptionForm =()=>{
+const[adoptionData, setAdoptionData] = useState({
+  name : "",
+  phoneNumber : "",
+  email: "",
+  homeStatus: "own"
+})
+let errorContainer = null
+const [errors, setErrors]=useState([])
```

Comment:
please add a space on either side of any equals signs, etc for readability. Here and on line 12 are two examples!